tests/test_utils.py

```diff
@@ -990,15 +990,49 @@ def test_hausdorff_distance(self):
         stats = ants.hausdorff_distance(s16, s64)
 
     def test_channels_first(self):
-        import ants
         image = ants.image_read(ants.get_ants_data('r16'))
         image2 = ants.image_read(ants.get_ants_data('r16'))
         img3 = ants.merge_channels([image,image2])
         img4 = ants.merge_channels([image,image2], channels_first=True)
-
         self.assertTrue(np.allclose(img3.numpy()[:,:,0], img4.numpy()[0,:,:]))
         self.assertTrue(np.allclose(img3.numpy()[:,:,1], img4.numpy()[1,:,:]))
-
+        
+    def test_polar_decomposition(self):
+        # Helper functions for creating known matrices
+        def make_known_rotation(theta_deg):
+            theta = np.deg2rad(theta_deg)
+            R = np.eye(3)
+            R[:2, :2] = [[np.cos(theta), -np.sin(theta)],
+                        [np.sin(theta),  np.cos(theta)]]
+            return R
+        def make_known_scaling_matrix(sx, sy, sz):
+            return np.diag([sx, sy, sz])        
+
+        # 1. Setup: Create a matrix from a known P and Z.        
+        # The key is to multiply them in the order P @ Z.
+        P_known = make_known_scaling_matrix(2.5, 1.0, 1.5)
+        Z_known = make_known_rotation(45) # Use Z for "orthogonal" part        
```

Comment:
Would recommend making the rotation not 45 degrees so that the ground truth is not symmetric

Reply:
Sure, I can add this in a subsequent pull request.